CONTRIBUTING.md

```diff
@@ -1,20 +1,31 @@
 ## Contributing
-GeoBlacklight is a collaborative open source projects where contributions are welcome. This contributing guide is borrowed in part from the [Hydra Contributing Guide](https://github.com/projecthydra/hydra/blob/master/CONTRIBUTING.md) and the [Blacklight Contributing Wiki](https://github.com/projectblacklight/blacklight/wiki/Contributing-to-Blacklight).
+GeoBlacklight is a collaborative open source projects where contributions are :sparkles:welcome:sparkles:. This contributing guide is borrowed in part from the [Samvera Contributing Guide](https://github.com/samvera/hydra/blob/master/CONTRIBUTING.md) and the [Blacklight Contributing Wiki](https://github.com/projectblacklight/blacklight/wiki/Contributing-to-Blacklight).
+
+### Who can contribute?
+**Anyone** is welcome to contribute to GeoBlacklight. We follow a set of contribution practices to maintain a technically sustainable and stable software project for everyone.
 
 ### Reporting issues
 Did you find a bug in GeoBlacklight or interested in a new feature? Make sure to add an issue for it in the [issue tracker](https://github.com/geoblacklight/geoblacklight/issues).
 
  - Make sure you have a [GitHub account](https://github.com/signup/free)
- - Submit a [Github issue](./issues) by:
+ - Submit a [GitHub issue](./issues) by:
     - Clearly describing the issue
     - Provide a descriptive summary
     - Explain the expected behavior
     - Explain the actual behavior
     - Provide steps to reproduce the actual behavior
 
+### GeoBlacklight Software Versioning
+GeoBlacklight follows the practice of [Semantic Versioning](https://semver.org/) for software releases. The declared semantically versioned public API includes:
+ - the GeoBlacklight-Schema
+ - the public GeoBlacklight Ruby codebase classes
+ - the GeoBlacklight JavaScript interface
+ - the GeoBlacklight view interface
 
 ### Contributing code or documentation
```

Comment:
This is excellent, and quite inviting to new contributors.  Might also referencing [generated code documentation](http://www.rubydoc.info/github/geoblacklight/geoblacklight/master/frames) be useful?  (I can offer a PR if so).

Reply:
Oh thats a great suggestion @jrgriffiniii . I'll update accordingly.